src/components/seekbar/seekbar.tsx

```diff
@@ -109,6 +111,13 @@ class SeekBar extends Component<any, any> {
         this.setState({resizing: false});
       }, Number(style.defaultTransitionTime));
     });
+    if (this.props.guiClientRect.width <= PLAYER_BREAK_POINTS.SMALL) {
```

Comment:
you shouldn't need to check the dimensions in a component explicitly, the active breakpoint should already be updated in state.shell.playerSize

also there's a state called shell.isMobile which might be relevant here as well, depending on what behavior we want on non-tiny mobile screens